discovery-provider/src/queries/get_health.py

```diff
@@ -271,11 +275,19 @@ def get_health(args, use_redis_cache=True):
         if error:
             return health_results, error
 
-    # Return error on unhealthy block diff if requested.
-    if enforce_block_diff and health_results["block_difference"] > healthy_block_diff:
-        return health_results, True
+    is_unhealthy = False
```

Comment:
very much a preference / nit, but occurs to me this could be written cleaner as:
```
unhealthy_block = enforce_block_diff and ...
unhealthy_challenges = challenge_events_age_max_drift and ...
is_unhealthy = unhealthy_block or unhealthy_challenges
```
this code will stretch better with this pattern as we add more of these checks I think?